### ISSUE TITLE
BlockColors crash if another mod has an error

### ISSUE THREAD

**Issue by WenXin20** [Ladysnake/Illuminations-for-Forge#2]
`Minecraft v1.14.4`
`Forge v28.1.109`
`Illuminations v0.1.0`

- If another mod has an error, such as with an invalid config file, Illuminations will crash instead of displaying the forge error screen and what went wrong with the other mod/config. 
- This could mislead the player into thinking Illuminations is at fault instead of something else.
- [Crash Report](https://gist.github.com/WenXin20/876a331d7a525a27967c611b16601a1b)

**Comment by AbsolemJackdaw** [comment]
thanks for the heads up.
i'm suspecting the direct call into minecraft's block registry to retreive the grass color for firefly grass coloring may be the culprit as these blocks may be skipped due to errors and are null as a result.
will look into this !

**Comment by WenXin20** [comment]
Yeah, I had to go through a lot of mods with this type of crash. It's a forge bug, but null checks can be added as a workaround to avoid it.
